src/common/librouter/test/usock_epipe.c

```diff
@@ -99,9 +100,12 @@ static void server_error_cb (struct usock_conn *conn, int errnum, void *arg)
 static void server_close_cb (struct usock_conn *conn, void *arg)
 {
     struct test_params *tp = arg;
+    tp->ready = 1;
```

Comment:
I'm rusty too, so might be wrong, but should `tp->ready` be modified with the mutex held?

Reply:
Heh, yeah. duh. :roll_eyes:

Reply:
Mind if I fix, squash, and force-push?

Reply:
Sure!